sdk/lib/cmlib/cmlib.h

```diff
@@ -500,6 +532,34 @@ CmPrepareHive(
 
 /* NT-style Public Cm functions */
 
+//
+// Check Registry Routines
+//
+BOOLEAN
+NTAPI
+CmpValidateSDsOfHive(
```

Comment:
If this function is going to be used only within cmcheck.c, then it doesn't need to be present in this global header.